localstack/services/cloudformation/provider.py

```diff
@@ -107,11 +116,12 @@ def __init__(self, metadata=None):
 
 
 class Stack:
-    def __init__(self, metadata=None, template=None):
+    def __init__(self, metadata=None, template=None, template_body=None):
         if template is None:
             template = {}
         self.metadata = metadata or {}
         self.template = template or {}
+        self.template_body = template_body
         self._template_raw = clone_safe(self.template)
```

Comment:
Actually, we had a similar intention when introducing `_template_raw`, which is supposed to represent a immutable version of the raw/original template we've received with the request. I guess we should be able to unify the two now and potentially `_template_raw` in this case..?

Reply:
I currently try to avoid touching existing concepts/fields wherever possible, instead introduce parallel concepts and slowly replace older parts with newer correct alternatives (in this case the template_body). Makes refactoring this just much easier because otherwise I'll have to deal with all the existing code paths which is exactly what I'm trying to avoid :grin: